Gruntfile.js

```diff
@@ -199,6 +199,12 @@ module.exports = function(grunt) {
       }
     },
     exec: {
+      apiDev: {
+        cmd: './node_modules/.bin/nodemon --watch api api/server.js'
+      },
+      sentinelDev: {
+        cmd: './node_modules/.bin/nodemon --watch sentinel sentinel/server.js'
+      },
```

Comment:
Consider using the grunt `watch` module which we use for webapp changes. I'm not sure if nodemon is better/worse but if it's equivalent it'd be better to use the same module for this too.

Reply:
I looked into it but it seems like `watch` isn't great for itself watching long running tasks, as there's no easy way to stop the server on a file change and then start it up again. Everyone recommends using `grunt-nodemon` instead (and here we're just using plain old `nodemon`)